### PR TITLE
fix: browser doesn't remember my color scheme preference

### LINKED ISSUE #386: Problem: browser doesn't remember my color scheme preference
# Problem
When I refresh browser it defaults to light color scheme, even if I selected dark one before the refresh

# Solution
Two ACs: 
- take into account user's OS preference setting https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
- when the UI theme switcher is used, remember the setting in the browser cookie (or similar)

To be resolved in a separate PR (not in this one).

_Originally posted by @zolotokrylin in https://github.com/holdex/holdex-venture-studio/issues/381#issuecomment-1591708302_

### PR DESCRIPTION
**Pull request by power-f-GOD**
This PR fixes #386.

Summary of the changes made:

1. [feat: Browser now remembers user previous theme preference.](https://github.com/holdex/holdex-venture-studio/commit/2bdb0f65dbb504392328b6121459372e5d152c53)
2. [fix: FODT (Flash Of Default Theme)](https://github.com/holdex/holdex-venture-studio/commit/56f6b34faf3537c2e33d95a25e49fc13a2bede23) 
    - Ditch `cookie` implementation for `localStorage` and remove `universal-cookie` dependency;
    - Use string literal to embed the initial theme `script` in the `head` element which should run before all other scripts;
    - Set the `data-theme` attribute on the root element, `document.documentElement`, instead of `document.body` since the same is `null` at the instant the embedded `script` is executed.

PS. The reason I went with the `localStorage` approach was that it was easier to get the specific item, (`theme`, in this case), from the storage which was necessary for the embedded `script` string. It wouldn't have been possible to do with cookies (or should I say it would have been more difficult, since I would have to parse the cookie string with some helper function and then extract the key value I needed which is apparently an overhead).📝

### CODE REVIEW

**Comment by power-f-GOD** [comment]
PR created.✔️

@georgeciubotaru , kindly review.🤲🏼

Cc: @zolotokrylin , @markholdex

File: src/app.html [hunk 1]
```diff
@@ -1,115 +1,49 @@
...
 
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap"
-      rel="stylesheet"
-    />
-    %sveltekit.head%
-  </head>
+<head>
```
Comment (zolotokrylin) [inline src/app.html:4]: how comes the `<head>` is not indented while it is a child element? strange a bit

Comment (zolotokrylin) [reply src/app.html:4]: other than that – looks good!

**Review by zolotokrylin — APPROVED** [review]

**Comment by power-f-GOD** [comment]
@surajsharma , works for me.✅ Seems the JS on the URL is taking some time to load and be executed.💯

**Comment by surajsharma** [comment]
@power-f-GOD considerable lag on localhost while toggling too, this must be looked into

**Pull request merged**